journal-of-zoo-biology.csl

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Journal of Zoo Biology</title>
+    <title-short>JZB</title-short>
+    <id>http://www.zotero.org/styles/journal-of-zoo-biology</id>
+    <link href="http://www.zotero.org/styles/journal-of-zoo-biology" rel="self"/>
+    <link href="http://www.zotero.org/styles/evolution" rel="template"/>
+    <link href="http://www.escijournals.net/JZB/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>O'Brien Patrick, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <updated>2017-01-24T14:20:42+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor-translator">
+    <names variable="editor translator" delimiter=", " prefix="(" suffix=")">
+      <name and="text" initialize-with=". " delimiter=", "/>
+      <label form="short" prefix=", " strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" initialize-with=". " name-as-sort-order="first"/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
```

Comment:
this doesn't seem like a good choice -- I'd add a separate title-short macro, using the short title and, more importantly, doesn't include the edition information. Then also add it to the title macro.